graph/kv/quadstore.go

```diff
@@ -272,6 +272,16 @@ func (qs *QuadStore) ValuesOf(ctx context.Context, vals []graph.Value) ([]quad.V
 	}
 	return out, last
 }
+
+// TODO: Proper BatchQuadStore support for this function
```

Comment:
May I ask you to implement it at least partially? :)

Check the `ValueOf`, an algorithm is quite simple: open `View` once, iterate over values inside, call `resolveQuadValue` for each and store the result. But instead of dropping an error as `ValueOf` does, return it here.